src/Tracy/templates/panel.phtml

```diff
@@ -0,0 +1,85 @@
+<?php
```

Comment:
It looks like this file is weird formatted, don't you think? 

![image](https://user-images.githubusercontent.com/538058/94460083-83b1a600-01b8-11eb-8bfb-ece0025da9af.png)

Reply:
yep, changed all to tabs.